src/extension/monitor/hibernate/src/main/java/org/geoserver/monitor/hib/EntityManagerFactoryPostProcessor.java

```diff
@@ -42,16 +44,16 @@ public Object postProcessBeforeInitialization(Object bean, String beanName)
     
     void init(AbstractEntityManagerFactoryBean factory) {
         try {
-            File f = data.findDataFile("monitoring", "hibernate.properties");
-            if (f == null) {
+            Resource f = data.get(Paths.path("monitoring", "hibernate.properties"));
+            if (!Resources.exists(f)) {
                 //copy one out from 
                 Properties props = new Properties();
                 props.putAll(factory.getJpaVendorAdapter().getJpaPropertyMap());
                 props.putAll(factory.getJpaPropertyMap());
                 
-                File monitoring = data.findOrCreateDataDir("monitoring");
-                File file = new File( monitoring, "hibernate.properties" );
-                FileOutputStream fout = new FileOutputStream(file);
+                Resource monitoring = data.get("monitoring");
+                Resource file =  monitoring.get("hibernate.properties" );
```

Comment:
Extra space before final closing bracket. Yes, it was there before, but feel free to preen lines you touch.  :-)